src/LanguageHelper.php

```diff
@@ -33,6 +35,52 @@ public function exists($lang, $basePath)
 	}
 
 	/**
+	 * Searches for a specific localise file for a given language. Falls back to the .
+	 *
+	 * @param   string  $lang      Language to check.
+	 * @param   string  $basePath  Base path to the language folder.
+	 *
+	 * @return  LocaliseInterface  The localise helper object.
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function findLocalise($lang, $basePath)
+	{
+		// Look for a language specific localise class
+		$class = str_replace('-', '_', $lang . 'Localise');
+		$paths = array();
+
+		$paths[0] = $basePath . "/overrides/$lang.localise.php";
+		$paths[1] = $basePath . "/$lang/$lang.localise.php";
+
+		ksort($paths);
+		$path = reset($paths);
+
+		while (!class_exists($class) && $path)
+		{
+			if (file_exists($path))
+			{
+				require_once $path;
+			}
+
+			$path = next($paths);
+		}
+
+		// If we have found a match initialise it and return it
+		if (class_exists($class))
```

Comment:
Add a check here to make sure the interface is implemented.  `if (class_exists($class) && $class instanceof LocaliseInterface)`